Integrations/integration-cisco-meraki.yml

```diff
@@ -38,12 +41,10 @@ script:
         return toReturn;
     };
 
-    // ------------------------ mics -------------------- //
-
     /**
        * Converts a demisto table in JSON form to a Markdown table
        * @param {String} name - the name of the table
-       * @param {Object} table - the JSON table - Array of objects with the same keys
+       * @param {Object | Array<Object>} table - the JSON table - Array of objects with the same keys
```

Comment:
Also look like a function for general use (and a well written one, :+1:) so please move to common